zenodo/config.py

```diff
@@ -262,6 +263,8 @@ def _(x):
     consumer_secret="CHANGE_ME",
 )
 
+OAUTH2_PROVIDER_TOKEN_EXPIRES_IN = 3600
```

Comment:
It's undefined because OAuth2Server is not installed on the API app - see https://github.com/inveniosoftware/invenio-oauth2server/pull/39